service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleServiceImpl.java

```diff
@@ -45,9 +46,13 @@ public MetaDbSample saveSampleMetadata(MetaDbSample
     public MetaDbSample setUpMetaDbSample(MetaDbSample metaDbSample) throws Exception {
         SampleMetadata sampleMetadata = metaDbSample.getLatestSampleMetadata();
         metaDbSample.setSampleClass(sampleMetadata.getTumorOrNormal());
-        metaDbSample.setPatient(new MetaDbPatient(sampleMetadata.getCmoPatientId()));
         metaDbSample.addSample(new SampleAlias(sampleMetadata.getIgoId(), "igoId"));
         metaDbSample.addSample(new SampleAlias(sampleMetadata.getInvestigatorSampleId(), "investigatorId"));
+
+        MetaDbPatient patient = new MetaDbPatient(sampleMetadata.getCmoPatientId());
+        patient.addPatientAlias(new PatientAlias(patient.getCmoPatientId(), "igoId"));
```

Comment:
Whats going to be the name of the property on the node igoId or cmoPatientId?

Reply:
igoId because of its namespace